Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/branch.py

```diff
@@ -71,19 +71,80 @@ def editable(cls, branch, repository=None):
 
     @classmethod
     def branch_point(cls, repository, limit=None):
+        """Find the closest parent which is on a non-editable branch"""
+        if not isinstance(repository, local.Git):
+            sys.stderr.write("Can only find the branch_point of a git repository\n")
+            return 1
```

Comment:
I'm not sure this is actually acceptable here, but we probably need to wait for @JonWBedard to weigh in on that.

Reply:
That's fine, this function is only ever invoked from `git` anyways.